components/folioform/LongAnswerCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Card, Textarea } from '@chakra-ui/react'
+
+interface LongAnswerCardProps {
+  label: string
+  prompt: string
+}
+
+const LongAnswerCard: React.FC<LongAnswerCardProps> = ({ label, prompt }) => {
+  return (
+    <Card
+      display='flex-column'
```

Comment:
Great job on the form inputs! But perhaps we could use more standard naming for form inputs e.g. `LongTextarea`?